csuntechlab/iSTART
# STR-272 -- Fix Wellness Goal Template

# Description
Refactored the Wellness Goal template & added prevention functionality of multiple emails being sent from user. Added loading screen. Adjusted wording for wellness goal being sent out.
  
# Testing
1. Visit the wellness goal slide and confirm all past functionality works.
2. Make sure the loading screen appears and prevents further interaction until the email is sent.
3. Ensure the `email wellness goal` button is now gone after sending a wellness goal out.
